test/helper.py

```diff
@@ -0,0 +1,42 @@
+
+import json, os
+from TestRunner import convert_to_snake_case
+
+
+def generate_test_cases(transformation, src_sentences=[], print_result=False, output_filename=""):
+    """
+    Sample code to demonstrate usage that can also assist in adding test cases.
+    """
```

Comment:
Minor fix: It would be good to expand the docstring with param details.

Reply:
Fixed, thank you for your highlighting this point!

I also fixed a problem with the directory path of the result JSON file, which was set to `os.path.dirname(os.path.realpath(__file__))` inside `generate_test_cases()`. Because this method was moved to another directory, I changed the path to absolute, so the caller have to specify the exact location of the output file and it should  work fine now.